app/controllers/users_controller.rb

```diff
@@ -48,4 +49,8 @@ def photo_galleries
     @galleries = @user.photo_galleries
   end
 
+  def latest_photo_comments
+    @latest_photo_comments = @user.latest_photo_comments.limit(10)
+  end
+
```

Comment:
I think the best approach is a helper function `latest_photo_comments(user)` which just returns the 10 latest photo comments.